packages/shorebird_cli/test/src/executables/patch_executable_test.dart

```diff
@@ -132,44 +132,47 @@ void main() {
       );
     });
 
-    group('when running on windows and process exits with code -1073741515', () {
+    group('when on windows', () {
       setUp(() {
-        const stdout = 'uh oh';
-        const stderr = 'oops something went wrong';
-        when(() => patchProcessResult.exitCode).thenReturn(-1073741515);
-        when(() => patchProcessResult.stderr).thenReturn(stderr);
-        when(() => patchProcessResult.stdout).thenReturn(stdout);
-
-        when(
-          () => shorebirdProcess.run(
-            any(that: endsWith('patch')),
-            any(),
-            runInShell: any(named: 'runInShell'),
-          ),
-        ).thenAnswer((_) async => patchProcessResult);
-
         when(() => platform.isWindows).thenReturn(true);
       });
-
-      test('throws a missing C++ runtime exception', () async {
-        await expectLater(
-          () => runWithOverrides(
-            () => patchExecutable.run(
-              releaseArtifactPath: 'release',
-              patchArtifactPath: 'patch',
-              diffPath: 'diff',
+      group('when the process exits with code -1073741515', () {
```

Comment:
These should be inverted imo. The group for the process exit code should be first and the windows group should be nested so you can test both the windows and non-windows behavior for the exit code.

Reply:
Hum, but does it makes sense? This error is specific to windows...What am I missing? 😅

Reply:
I think Felix is suggesting that we should ensure that this path doesn't trigger for non-Windows users and that the error is otherwise handled gracefully

Reply:
Got it! Updated, please lmk if I understood it correctly